github/AuthenticatedUser.py

```diff
@@ -973,9 +973,11 @@ def get_notifications(
 
         params = dict()
         if all is not github.GithubObject.NotSet:
-            params["all"] = all
+            # convert True, False to true, false for api parameters
+            params["all"] = str(all).lower() if isinstance(all, bool) else all
         if participating is not github.GithubObject.NotSet:
-            params["participating"] = participating
+            # convert True, False to true, false for api parameters
+            params["participating"] = str(participating).lower() if isinstance(participating, bool) else participating
```

Comment:
At this point, you know `participating` is set and is a `bool`, so I think you could be simplified to:

`params["participating"] = "true" if participating else "false"`

Reply:
And for `all` as well

Reply:
Okay sounds good.